### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by Henrynunez112**
**Description**

Second Draft of the Widget Stats

*Mobile*
Screen Shot 2022-01-27 at 1 49 49 PM

*Desktop*
Screen Shot 2022-01-27 at 1 42 46 PM

issue #552

### CODE REVIEW

File: src/home/stats/Stats.tsx [hunk 1]
```diff
@@ -83,24 +118,43 @@ export const Datum = memo<DatumProps>(function _Datum({ value, title, id, link }
...
-    </>
+    </div>
   )
 })
 
+const spanBodyCss = css(flex, {
+  flexDirection: "row",
+  alignItems: "center",
+  justifyContent: "center",
+  ["@media (max-width: 1165px)"]: {
```
Comment (aaronmgdr) [inline src/home/stats/Stats.tsx:129]: can you use one of our standard `WHEN_*` constants here? and in other similar places?

File: src/home/stats/Stats.tsx [hunk 2]
```diff
@@ -30,32 +33,64 @@ const rootCss = css(flex, {
...
+  backgroundColor: colors.dark,
+  borderRadius: 16,
+  width: 724,
+  height: 116,
+  padding: "24px 40px",
   zIndex: 20,
   ["@media (max-width: 1165px)"]: {
-    display: "none",
+    width: 260,
+    height: 222,
```
Comment (aaronmgdr) [inline src/home/stats/Stats.tsx:46]: really not a good idea to set both width and height of something containing text.so many ways for that to get messed up. 

Example
* different language
* numbers get bigger
* px density or someone manuall resizes tx

Comment (aaronmgdr) [reply src/home/stats/Stats.tsx:46]: better practice is to set neight an reliy on padding alone to get to a implicit size

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
see other comments

File: src/home/stats/Stats.tsx (hunk 2 above)
Comment (Henrynunez112) [reply src/home/stats/Stats.tsx:47]: So many factors to think about!

File: src/home/stats/Stats.tsx [hunk 3]
```diff
@@ -42,15 +42,16 @@ const rootCss = css(flex, {
   height: 116,
   padding: "24px 40px",
   zIndex: 20,
+  boxShadow: "0px 1px 16px rgba(255, 255, 255, 0.16), 0px 0px 24px rgba(78, 236, 255, 0.6)",
   [WHEN_MOBILE]: {
     width: 260,
-    height: 222,
+    // height: 222,
     padding: "24px 0px",
+    paddingBottom: 222,
```
Comment (aaronmgdr) [inline src/home/stats/Stats.tsx]: wow thats a lot

File: src/home/stats/Stats.tsx [hunk 4]
```diff
@@ -30,32 +34,63 @@ const rootCss = css(flex, {
...
-  borderRadius: 6,
-  right: 0,
-  top: 210,
-  padding: 24,
-  paddingBottom: 30,
+  flexDirection: "column",
+  backgroundColor: colors.dark,
+  borderRadius: 16,
+  width: 724,
+  height: 116,
```
Comment (aaronmgdr) [inline src/home/stats/Stats.tsx:42]: set a max width and remove height and width.

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**